### ISSUE TITLE
slide deck to introduce databases

### ISSUE THREAD

**Issue by azayneeva** [foundersandcoders/master-reference#778]
I've created a slide deck to introduce databases. It was useful to give students a verbal explanation of the topic before we started working on workshops.

If it's found useful in other campuses this can be added to the schedule.

https://docs.google.com/presentation/d/1Fo1R-dNZAxkG2KUmelJc7UcpBMy-RuPDxdwGIoK6mEc/edit?usp=sharing

**Comment by shiryz** [comment]
@azayneeva are these based on Steve's slides?
If all campuses are going to do an introduction (naz will) I think we should add the slides to [this repo](https://github.com/foundersandcoders/intro-to-databases) (probably with some modification to the readme) and get it merged into the schedule.

**Comment by azayneeva** [comment]
@shiryz yes, it's heavily based on Steve's slides.
What should I change before merging it?

**Comment by shiryz** [comment]
@azayneeva I think the only missing is some explanation about primary and foreign keys and I believe the numbers don't match anymore.

**Comment by ZooeyMiller** [comment]
In general from a quick scan of the slides they look ok to me, but I think @shiryz is right, we should be working towards improving [foundersandcoders/intro-to-databases](https://github.com/foundersandcoders/intro-to-databases) and put any materials related to it in there.

**Comment by azayneeva** [comment]
@shiryz  we've created a [pr](https://github.com/foundersandcoders/intro-to-databases/pull/13)  :)

**Comment by astroash** [comment]
@eliascodes, the [above mentioned repo](https://github.com/foundersandcoders/intro-to-databases) is not linked anywhere in the week-6 README.md. I don't think this issue should be closed until this happens as it's relying on alumni knowledge of it's existence, which can easily be lost.

**Comment by eliascodes** [comment]
@astroash PR welcome

**Comment by astroash** [comment]
Sweet, although there’s currently a 2 month lead time on my to-do list
